(DOCSP-25159) [BAAS] Access App metadata with function context (#187)
mongodb/docs-app-services: source/functions/context.txt

   * - :ref:`context.values <context-values>`
     - Contains static global :ref:`values <values-and-secrets>`.

.. _context-app:

Get App Metadata (``context.app``)
----------------------------------

> The ``context.app`` object contains metadata about the Atlas App that

Reviewer: ```suggestion
The ``context.app`` object contains metadata about the App that
```

contains the function.

.. code-block:: typescript
   
   {
     "projectId": string,
     "id": string,
     "name": string,
     "clientAppId": string,
     "deployment": {
       "providerRegion": string,
       "model": string,
     },
     "lastDeployed": string,
     "hostingUri": string,
   }

.. method:: context.app.id
   

>    The ID of the Atlas App the contains the function.

Reviewer: [typo] "the" should probably be "that" and update to just "App"
```suggestion
   The ID of the App that contains the function.
```

   .. example::
      
      .. code-block:: json
         
         "60c8e59866b0c33d14ee634a"

.. method:: context.app.projectId

>    The ID of the Atlas Project/Group that contains the App.

Reviewer: ```suggestion
   The :ref:`ID of the Atlas Project/Group <find-your-project-or-app-id>` that contains the App.
```

   .. example::
      
      .. code-block:: json
         
         "5e1ec444970199272441a214"

.. method:: context.app.name

>    The name of the Atlas App that contains the function.

Reviewer: ```suggestion
   The name of the App that contains the function.
```

   .. example::
      
      .. code-block:: json
         
         "myapp"

.. method:: context.app.clientAppId
   

>    The unique App ID for the Atlas App that contains the function.

Reviewer: ```suggestion
   The unique App ID for the App that contains the function.
```

   
   .. example::
      
      .. code-block:: json
         
         "myapp-abcde"

.. method:: context.app.deployment
   
   An object that describes the App's :ref:`deployment model and region <deployment-model>`.
   
   .. example::
      
      .. code-block:: json
         
         {
           "providerRegion": "aws-us-east-1",
           "model": "GLOBAL"
         }

.. method:: context.app.lastDeployed
   

>    The date and time that the Atlas App was last deployed formatted as an ISODate string.

Reviewer: ```suggestion
   The date and time that the App was last deployed, formatted as an ISODate string.
```

   
   .. example::
      
      .. code-block:: javascript
         
         "2022-10-31T12:00:00.000Z"

.. method:: context.app.hostingUri
   

>    The base URL of :ref:`static hosting <static-hosting>` assets, if enabled.

Reviewer: Style guide says we should put conditionals at the beginning of a sentence so readers can skip it if it doesn't apply.
```suggestion
   If :ref:`static hosting <static-hosting>` is enabled, this value is the base URL for assets.
```
